web/src/app/chat/message/Resubmit.tsx

```diff
@@ -56,17 +57,20 @@ export const ErrorBanner = ({
           )}
           {stackTrace && (
             <div className="mt-2 border-t border-neutral-200 dark:border-neutral-700 pt-2">
-              <button
-                onClick={() => setIsStackTraceExpanded(!isStackTraceExpanded)}
-                className="flex items-center gap-1.5 text-xs text-neutral-600 dark:text-neutral-400 hover:text-neutral-800 dark:hover:text-neutral-200 cursor-pointer"
-              >
-                {isStackTraceExpanded ? (
-                  <ChevronDown className="h-3 w-3" />
-                ) : (
-                  <ChevronRight className="h-3 w-3" />
-                )}
-                Stack trace
-              </button>
+              <div className="flex flex-1 justify-between">
+                <button
+                  onClick={() => setIsStackTraceExpanded(!isStackTraceExpanded)}
+                  className="flex items-center gap-1.5 text-xs text-neutral-600 dark:text-neutral-400 hover:text-neutral-800 dark:hover:text-neutral-200 cursor-pointer"
+                >
+                  {isStackTraceExpanded ? (
+                    <ChevronDown className="h-3 w-3" />
+                  ) : (
+                    <ChevronRight className="h-3 w-3" />
+                  )}
+                  Stack trace
+                </button>
```

Comment:
Just a small nit, not sure if you want to tackle it in this PR or not:

Could we please switch over to use the `refresh-components/buttons/Button` component instead, please? The implementation would be very simple, and would actually delete most of these ugly, custom stylings:

```tsx
<Button icon={isStackTraceExpanded ? SvgChevronDown : SvgChevronUp} onClick={..} >Stack Trace</Button>
```

I know it wasn't your addition, but if we're touching this section, might as well clean it up a bit as well. If you want to open up a new, follow-up PR to tackle this, then that's fine too.

Reply:
If you think the `primary` variant is too ugly, then feel free to use the `secondary`, `tertiary`, or `internal` variants as well.

```tsx
<Button secondary icon={isStackTraceExpanded ? SvgChevronDown : SvgChevronUp} onClick={..} >Stack Trace</Button>
```

Reply:
Sounds good. Resolving this thread.